src/fides/api/oauth/utils.py

```diff
@@ -187,6 +187,23 @@ async def verify_oauth_client(
     return client
 
 
+async def extract_client_id(
+    authorization: str,
+    db: Session = Depends(get_db),
+) -> ClientDetail:
+    """
+    Verifies that the access token provided in the authorization header contains
+    the necessary scopes or roles specified by the caller. Yields a 403 forbidden error
+    if not.
+
+    NOTE: This function may be overwritten in `main.py` when changing
```

Comment:
nit - looks like this may have been copy pasta'd from the `verify_oauth_client` doc string, and it doesn't apply to this function. is that right?

Reply:
Correct 👍🏽

Reply:
I _think_ I am able to just remove it completely, tested well for me just so now so am going to just go straight to `extract_token_and_load_client` instead